### PR TITLE
feat: move workers queries to read replica 1

### PR DESCRIPTION
**Pull request by capJavert**
Moved workers:
- digest generation
- all workers not needing real time data (upvotes, rep, notification with counts)

Did not move any worker that needs to write or live data as well. Some of them I implemented but in testing realized they would return stale data since RR would be out of date. 

We could move rest of the real time workers if we moved CDC to RR, see [here](https://dailydotdev.slack.com/archives/C03C3JZPRU4/p1733484456135089), but something to be evaluated separate.

### CODE REVIEW

File: src/common/feedGenerator.ts [hunk 1]
```diff
@@ -307,7 +308,7 @@ const sourcesToFilters = ({
 };
 
 export const feedToFilters = async (
-  con: DataSource,
+  con: DataSource | EntityManager,
```
Comment (capJavert) [inline src/common/feedGenerator.ts:311]: Main change for digest to work, in digest we inject `queryRunner.manager` from `queryReadReplica`.

File: src/common/personalizedDigest.ts [hunk 2]
```diff
@@ -231,11 +232,13 @@ export const getPersonalizedDigestEmailPayload = async ({
   previousSendDate: Date;
   feature: PersonalizedDigestFeatureConfig;
 }): Promise<SendEmailRequestWithTemplate | undefined> => {
-  const feedConfig = await feedToFilters(
-    con,
-    personalizedDigest.userId,
-    personalizedDigest.userId,
-  );
+  const feedConfig = await await queryReadReplica(con, ({ queryRunner }) => {
```
Comment (rebelchris) [inline src/common/personalizedDigest.ts:235]: Yeah here it should be fine, the feed one is different story I guess, this one can be async/outdated even.

**Review by rebelchris — APPROVED** [review]
Looks good to me.
Don't see any issues on any of these, but let's monitor after deployment for a bit. 

Nice we have this function 😂

File: src/common/personalizedDigest.ts (hunk 2 above)
Comment (omBratteng) [inline src/common/personalizedDigest.ts:235]:
```suggestion
  const feedConfig = await queryReadReplica(con, ({ queryRunner }) => {
```

**Review by omBratteng — APPROVED** [review]

**Pull request merged**